src/net_processing.cpp

```diff
@@ -517,7 +517,8 @@ bool AddOrphanTx(const CTransactionRef& tx, NodeId peer) EXCLUSIVE_LOCKS_REQUIRE
     // 100 orphans, each of which is at most 99,999 bytes big is
     // at most 10 megabytes of orphans and somewhat more byprev index (in the worst case):
     unsigned int sz = tx->GetTotalSize();
-    if (sz >= MAX_STANDARD_TX_SIZE) {
+    unsigned int nMaxSize = tx->IsShieldedTx() ? MAX_TX_SIZE_AFTER_SAPLING : MAX_STANDARD_TX_SIZE;
+    if (sz >= nMaxSize) {
```

Comment:
Do we want to reduce `DEFAULT_MAX_ORPHAN_TRANSACTIONS` to 25, to compensate this change?
I don't think that we can reach 100 orphan txes at any time (given the faster block time, and the low avg number of txes), and the user can customize it with `-maxorphantx` anyway.

In any case, we should update the comment.

Reply:
Updated with both changes.